src/v2/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -251,6 +252,13 @@ const TrackingWrappedArtworkApp: React.FC<Props> = props => {
   const referrer = state && state.previousHref
   const { isComplete } = useRouteComplete()
 
+  // Websocket setup for extended bidding auctions.
+  const enabled = !!sale?.extendedBiddingIntervalMinutes
+  const channelInfo = {
+    channel: "SalesChannel",
+    sale_id: sale?.internalID,
+  }
```

Comment:
maybe move this object directly into the context below so that it feels more contained